tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/source_assets.py

```diff
@@ -63,7 +63,11 @@ def _copy_file(n: int, dataset: Dataset, asset_dir: str, N: int) -> str:
         raise last_error
 
     # verify file size is as expected, if we know the size a priori
-    assert (dataset.asset_h5ad_filesize == -1) or (dataset.asset_h5ad_filesize == os.path.getsize(dataset_path))
+    assert (dataset.asset_h5ad_filesize == -1) or (
```

Comment:
added better message for assert.  Change is unrelated to this PR - tripped over this during my testing, which overlapped with the schema four migration, which in turn triggered this assert to fail.